src/main/java/ch/njol/skript/ScriptLoader.java

```diff
@@ -380,95 +368,95 @@ public final static ScriptInfo loadScripts(final List<Config> configs, final Lis
 			logHandler.printLog(); // Won't print anything, but handler is properly closed
 		}
 	}
-	
+
 	/**
 	 * Loads the specified scripts.
-	 * 
+	 *
 	 * @param configs Configs for scripts, loaded by {@link #loadStructure(File)}
 	 * @return Info on the loaded scripts
 	 */
 	@SuppressWarnings("null")
-	public final static ScriptInfo loadScripts(final Config... configs) {
+	public static ScriptInfo loadScripts(final Config... configs) {
 		return loadScripts(Arrays.asList(configs));
 	}
-	
+
 	/**
 	 * Load specified scripts.
-	 * 
+	 *
 	 * @param files Script files.
 	 * @return Info on the loaded scripts.
 	 * @deprecated Use the methods that take configs as parameters.
 	 */
 	@Deprecated
-	public final static ScriptInfo loadScripts(final File... files) {
+	public static ScriptInfo loadScripts(final File... files) {
 		List<Config> configs = loadStructures(files);
 		return loadScripts(configs);
 	}
-	
+
 	/**
 	 * Represents data for event which is waiting to be loaded.
 	 */
 	private static class ParsedEventData {
-		
-		public ParsedEventData(NonNullPair<SkriptEventInfo<?>, SkriptEvent> info, String event, SectionNode node, List<TriggerItem> items) {
+		ParsedEventData(NonNullPair<SkriptEventInfo<?>, SkriptEvent> info, String event, SectionNode node, List<TriggerItem> items) {

```

Comment:
A class, no matter if inner class, should have first line free. Improves readibility.